tests/unit/Commands/SyncWoocommerceCategoriesTest.php

```diff
@@ -67,6 +68,15 @@ public function testRun()
         // Run the category import command
         $this->runner->execute(SyncWoocommerceCategories::getCommandName());
 
+        $termMeta = WordpressExceptionThrower::throwExceptionOnWpError(
+            get_option(StorekeeperSeo::STOREKEEPER_SEO_OPTION_KEY)
+        );
+        $this->assertEquals(
+            count($termMeta['product_cat']),
+            1,
+            'Only one category should have synchronized SEO data'
```

Comment:
can you check if it's also set on the correct category? and if the data is correct?